src/services/AbstractPalletsService.ts

```diff
@@ -262,6 +277,40 @@ export abstract class AbstractPalletsService extends AbstractService {
 		return palletItemMeta;
 	}
 
+	private getDispatchablesItemMeta(
+		palletMeta: PalletMetadataV14,
+		dispatchableItemMetaIdx: number,
+		dispatchableItemId: string
+	): [number, FunctionMetadataLatest] {
+		const palletName = stringCamelCase(palletMeta.name);
+		const dispatchables = this.api.tx[palletName];
+
+		if ((palletMeta.calls as unknown as PalletCallMetadataV14).isEmpty) {
+			throw new InternalServerError(
+				`No dispatchable items found in ${palletMeta.name.toString()}'s metadata`
+			);
+		}
+
+		for (const [, val] of Object.entries(dispatchables)) {
+			const item = val.meta;
+			console.log(item.name);
```

Comment:
```suggestion
```
This line should be removed otherwise all the values from the loop are logged in the terminal where sidecar is running.